network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/publication_hero.html

```diff
@@ -69,13 +69,13 @@ <h2 class="mt-4 body-large">
             </p>
           </div>
         {% endif %}
-        {% if page.publication_file %}
-          <a class="mt-3 btn btn-primary download-button" href="{{ page.publication_file.url }}" download>
+        {% if download_file %}
```

Comment:
Made the "publication_file" part of this template a little more robust by passing in a `download_file` object into the context. This way it can be used on Publication Pages and Article Pages.